test/issue.spec.js

```diff
@@ -2,20 +2,43 @@ import expect from 'must';
 
 import Github from '../lib/GitHub';
 import testUser from './fixtures/user.json';
+import wait from './helpers/wait';
 import {assertSuccessful} from './helpers/callbacks';
+import getTestRepoName from './helpers/getTestRepoName';
 
 describe('Issue', function() {
    let github;
+   const testRepoName = getTestRepoName();
    let remoteIssues;
 
-   before(function() {
+   before(function(done) {
```

Comment:
Mocha allows you to return a promise chain instead of passing the done callback.